src/period.c

```diff
@@ -262,6 +262,7 @@ void libxmp_c2spd_to_note(int c2spd, int *n, int *f)
 	*f = c % 128;
 }
 
+#ifndef LIBXMP_CORE_PLAYER
```

Comment:
This one might actually be needed by core (#357). @sagamusix should S3M in GUS mode be using this table?

Reply:
Probably not. You can observe a lower precision in the lower frequencies when using the GUS driver in ST3, but that can probably be implemented in a different way if you really want to emulate it (OpenMPT does for the time being).

Reply:
OK, then disregard this complaint. :D

Reply:
Typo corrected: OpenMPT *doesn't* emulate reduced GUS frequency precision. Either way, if the table would required for that, it can always be ifdeffed in again, right?

Reply:
Right.